news/2021-08-11-new-featured-artist-natsume-itsuki.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "New Featured Artist: Natsume Itsuki"
+date: 2021-08-11 09:00:00 +0000
+---
+
+Highly expressive singer and voice actor **Natsume Itsuki** joins the Featured Artist lineup!
+
+![](https://assets.ppy.sh/artists/189/header.jpg)
+
+Known for handling essentially all types of vocal atmospheres and moods, **Natsume Itsuki** is the Tokyo-based talent responsible for 1/3 of vocal trio [La prière](https://osu.ppy.sh/beatmaps/artists/174) and her own self-released music that intersects outstanding vocals and a variety of rhythm game styles.
+
+We've snagged just about **every song** (19 in total) from her 4 album discography, so check it all out from [her Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/189)!
```

Comment:
```suggestion
We've snagged just about **every song** (19 in total) from her 4-album-discography, so check it all out from [her Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/189)!
```

Reply:
it's either that or "discography of 4 albums". "album discography" isn't what is meant here (if it even is a real term, and even if it was, it would seem like it was mistakenly put into singular) and "4-album discography" doesn't quite work because "4-album" isn't a thing on its own.

Reply:
i think dashes are better, but if so i'd also change "4" to "four"

Reply:
did it